README.md

```diff
@@ -43,22 +43,22 @@ if __name__ == '__main__':
 ```
 
 ## Material-UI components ported to Dash
-- [x] [`AutoComplete`](http://www.material-ui.com/components/auto-complete)
+- [x] [`AutoComplete`](http://www.material-ui.com/components/autocomplete)
 - [x] [`BottomNavigation`](http://www.material-ui.com/components/bottom-navigation)
-- [x] [`Checkbox`](http://www.material-ui.com/components/checkbox)
-- [x] [`CircularProgress`](http://www.material-ui.com/components/circular-progress)
-- [x] [`Dialog`](http://www.material-ui.com/components/dialog)
-- [x] [`Divider`](http://www.material-ui.com/components/divider)
-- [x] [`Drawer`](http://www.material-ui.com/components/drawer)
-- [x] [`DropDownMenu`](http://www.material-ui.com/components/dropdown-menu)
-- [x] [`FlatButton`](http://www.material-ui.com/components/flat-button)
-- [x] [`FontIcon`](http://www.material-ui.com/components/font-icon)
-- [x] [`IconButton`](http://www.material-ui.com/components/icon-button)
+- [x] [`Checkbox`](http://www.material-ui.com/components/checkboxes)
+- [x] [`CircularProgress`](http://www.material-ui.com/components/progress)
+- [x] [`Dialog`](http://www.material-ui.com/components/dialogs)
+- [x] [`Divider`](http://www.material-ui.com/components/dividers)
+- [x] [`Drawer`](http://www.material-ui.com/components/drawers)
+- [x] [`DropDownMenu`](http://www.material-ui.com/components/menus)
+- [x] [`FlatButton`](http://www.material-ui.com/components/buttons)
+- [x] [`FontIcon`](https://material-ui.com/components/icons/#icon-font-icons)
+- [x] [`IconButton`](http://www.material-ui.com/components/icons/#semantic-svg-icons)
 - [x] [`Paper`](http://www.material-ui.com/components/paper)
-- [x] [`RaisedButton`](http://www.material-ui.com/components/raised-button)
-- [x] [`Snackbar`](http://www.material-ui.com/components/snackbar)
-- [x] [`Subheader`](http://www.material-ui.com/components/subheader)
-- [x] [`Toggle`](http://www.material-ui.com/components/toggle)
+- [x] [`RaisedButton`](http://material-ui.com/components/buttons/#contained-buttons)
+- [x] [`Snackbar`](http://www.material-ui.com/components/snackbars)
+- [x] [`Subheader`](http://material-ui.com/components/lists/#pinned-subheader-list)
+- [x] [`Toggle`](http://www.material-ui.com/components/toggle-button)
```

Comment:
This one is the old name, but is actually the [switch component](https://material-ui.com/components/switches/).

Reply:
RaisedButton?

Reply:
ah

Reply:
The Toggle.